scripts/googleAppScripts/SheetToGithubIssue.gs

```diff
@@ -6,38 +6,62 @@ function onFormSubmit(e) {
   var category = formResponse['קטגוריה'][0];
   var mainLink = formResponse['לינק לאתר או קישור ראשי'][0];
   var description = formResponse['תיאור פרטי היוזמה'][0];
-  var remarksForAdmins = formResponse['הערות למנהלי העמוד'][0];
   var whatsappLink = formResponse['Whatsapp link'][0];
   var telegramLink = formResponse['Telegram link'][0];
   var driveLink = formResponse['Google Drive link'][0];
-  var gFormLink = formResponse['Google Form link'][0];
-  var gDocLink = formResponse['Google Doc link'][0];
   var discordLink = formResponse['Discord link'][0];
   var instagramLink = formResponse['Instagram link'][0];
   var tiktokLink = formResponse['Tiktok link'][0];
   var twitterLink = formResponse['Twitter / X link'][0];
   var facebookLink = formResponse['Facebook link'][0];
 
-  // Construct the issue body using the form response data
-  var issueBody = "### Initiative Name\n" + initiativeName +
-                    "\n\n### Category\n" + category +
-                    "\n\n### Category\n" + category +
-                    "\n\n### Main Link\n" + mainLink +
-                    "\n\n### Description\n" + description +
-                    "\n\n### WhatsApp Link\n" + (whatsappLink || 'N/A') +
-                    "\n\n### Remarks for Admins\n" + (remarksForAdmins || 'N/A') +
-                    "\n\n### Telegram Link\n" + (telegramLink || 'N/A') +
-                    "\n\n### Drive Link\n" + (driveLink || 'N/A') +
-                    "\n\n### Form Link\n" + (gFormLink || 'N/A') +
-                    "\n\n### Doc Link\n" + (gDocLink || 'N/A') +
-                    "\n\n### Discord Link\n" + (discordLink || 'N/A') +
-                    "\n\n### Instagram Link\n" + (instagramLink || 'N/A') +
-                    "\n\n### TikTok Link\n" + (tiktokLink || 'N/A') +
-                    "\n\n### Twitter/X Link\n" + (twitterLink || 'N/A') +
-                    "\n\n### Facebook Link\n" + (facebookLink || 'N/A');
-
-    // Use the initiative name as the issue title
-    var issueTitle = initiativeName;
+  // todo: adjust the form and add these fields
+  var initiativeDetails = '_No response_';
+  var validationLink = '_No response_';
+  var websiteLink = '_No response_';
+  var phoneNumber = '_No response_';
+  var email = '_No response_';
+  var formLink = formResponse['Google Form link'][0];
+  var documentLink = formResponse['Google Doc link'][0];
+  var linkedinLink = '_No response_';
+  var youtubeLink = '_No response_';
+  var donationLink = '_No response_';
+  var logoLink = '_No response_';
+  var tags = '_No response_';
+
+  // Function to format the value or return "_No response_"
+  function formatValue(value) {
```

Comment:
I would say a more fitting name would be `NormalizeResponse` (and IMO you can delete the comment, reading the implementation is faster)